lib/index.js

```diff
@@ -63,7 +66,15 @@ import {ParseLatin} from 'parse-latin'
  * @param {Options | null | undefined} [options]
  * @returns {TransformMutate}
  *
- * @param {Parser | Processor} destination
+ * @overload
+ * @param {Parser | Processor} [destination]
```

Comment:
```suggestion
 * @param {Parser | Processor} destination
```

destination is not optional

Reply:
👍 Reverted.